cincoctrl/cincoctrl/users/models.py

```diff
@@ -89,13 +89,17 @@ class Repository(models.Model):
 
     class Meta:
         verbose_name_plural = "repositories"
+        ordering = ["name"]
 
     def __str__(self):
         return self.name
 
     def get_state_line(self):
         return f"{self.city}, {self.state} {self.zipcode}, {self.country}"
 
+    def description_no_space(self):
```

Comment:
Optional: The name `description_no_space` makes me think there will be no spaces, but it actually adds spaces if they are in the middle of the text. Probably low risk, but maybe worth renaming?

Reply:
you're right, updated to more appropriate name.